README.md

```diff
@@ -1 +1,125 @@
 # local-volume-provider
+
+what is it here...
+
+## Deploying the plugin
+
+To deploy the plugin image to a Velero server:
+
+1. Make sure your image is pushed to a registry that is accessible to your cluster's nodes.
+2. Run `velero plugin add <registry/image:version>`. Example with a ttl.sh image: `velero plugin add ttl.sh/dans/local-volume-provider`.
+
+### Customization
+
+You can configure certain aspects of plugin behavior by customizing the follow ConfigMap spec and adding to the Velero namespace.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: local-volume-provider-config
+  namespace: velero
+  labels:
+    velero.io/plugin-config: ""
+    replicated.com/nfs: ObjectStore
+    replicated.com/hostpath: ObjectStore
+data:
+  # Customize these values
+  veleroDeploymentName: velero
+  resticDaemonsetName: restic
+  fileserverImage: ttl.sh/dans/fileserver:2h
+```
+
+## Removing the plugin
+
+The plugin can be removed with `velero plugin remove replicated/local-volume-provider:latest`.
+This does not detach/delete any volumes that were used during operation.
+These can be removed manually using `kubectl edit` or by re-deploying velero (`velero uninstall` and `velero install ...`)
+
+## Usage Examples
+
+### HostPath
+
+```yaml
+apiVersion: velero.io/v1
+kind: BackupStorageLocation
+metadata:
+  name: default
+  namespace: velero
+spec:
+  backupSyncPeriod: 2m0s
+  provider: replicated.com/hostpath
+  objectStorage:
+    # This corresponds to a unique volume name
+    bucket: hostPath-snapshots
+  config:
+    # This path must exist on the host and be writable outside the group
```

Comment:
I was testing this but just setting `chmod -R 777` on the host or share, so I could re-phrase this as `all-user write access`. 

Maybe a better solution would be adding a `fsGroup` to the Velero pod to make it so you can scope the directly permissions to a particular group. Haven't really messed with this before, so input is appreciated.

Reply:
First version here probably doesn't need the fsGroup.  I'd have to double check that it's in there but any of these requirements for directory structure and permission requirements should be surfaced in the readme to avoid problems for the end-user

Reply:
Going to add to the discussion of file permissions. I also noticed that out current Minio solution has files created with a `fsGroup` and `runAsUser`, so I will add those as configurable plugin options.

Reply:
Done!